src/components/UserInfo/UserInfo.scss

```diff
@@ -0,0 +1,3 @@
+.UserInfo {
+  font-style: italic;
```

Comment:
Consider using more descriptive class names. `.UserInfo` is too generic and may cause naming conflicts with other components. 🤔